auter.yumdnfModule

```diff
@@ -19,17 +19,19 @@ function check_package_manager_lock() {
   [[ "${PACKAGEMANAGERLOCKWAITTIME}" -lt 1 ]] && PACKAGEMANAGERLOCKWAITTIME=60
 
   if [[ -f /var/run/yum.pid ]]; then
-    for LOCKCHECKATTEMPT in $(seq 1 $((PACKAGEMANAGERLOCKRETRIES + 1))); do
-      # If this has reached the retry limit configured then abort
-      if [[ ${LOCKCHECKATTEMPT} -eq $((PACKAGEMANAGERLOCKRETRIES + 1)) ]]; then
-        logit "ERROR: Final attempt to wait for yum to release lock file failed. Aborting auter run"
-        quit 3
-      fi
+    if { pgrep --pidfile /var/run/yum.pid >/dev/null ; } ; then
```

Comment:
Do you have any objections to the following? As we're only after the return code.
```suggestion
    if { pgrep --pidfile /var/run/yum.pid &>/dev/null ; } ; then
```

The same with line 32.

Reply:
no objections